source/strings.tex

```diff
@@ -1202,8 +1204,7 @@
 
 \pnum
 \postconditions
-\tcode{data()} is a non-null pointer that is copyable and can have \tcode{0} added
-to it, \tcode{size()} is \tcode{0}, and \tcode{capacity()} is an unspecified value.
+\tcode{capacity()} is an unspecified value.
```

Comment:
From where do we know that `size() == 0` here?

Reply:
Oh, yes, that should not have been removed. Fixed.